utils/aws_api.py

```diff
@@ -424,3 +424,19 @@ def get_users_keys(self):
     def get_user_keys(self, iam, user):
         key_list = iam.list_access_keys(UserName=user)['AccessKeyMetadata']
         return [uk['AccessKeyId'] for uk in key_list]
+
+    def get_support_cases(self):
+        self.support_cases = {}
```

Comment:
why `self`?

Reply:
for future reuse

Reply:
I would initialize it to `None` then, in the class

Reply:
remove it from self.